<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordRuleWidget</class>
 <widget class="QWidget" name="PasswordRuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>41</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="label_0">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Chaque</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="day_comboBox">
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentText">
      <string>Jour</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>Jour</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Lundi</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mardi</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mercredi</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Jeudi</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Vendredi</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Samedi</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dimanche</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>de</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTimeEdit" name="min_timeEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Ã </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTimeEdit" name="max_timeEdit">
     <property name="time">
      <time>
       <hour>23</hour>
       <minute>59</minute>
       <second>59</second>
      </time>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Attendre</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTimeEdit" name="delay_timeEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="remove_button">
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
